pkgs/io_file/lib/src/vm_posix_file_system.dart

```diff
@@ -13,6 +13,11 @@ import 'package:stdlibc/stdlibc.dart' as stdlibc;
 import 'file_system.dart';
 import 'internal_constants.dart';
 
+const _maxInt32 = 2147483647;
+
+/// The default `mode` to use with `open` calls that may create a file.
+const _defaultMode = 438; // => 0666 => rw-rw-rw-
```

Comment:
have you considered Lasse's idea how to represent permissions: https://github.com/dart-lang/language/issues/2708#issuecomment-1345359578 ?

Reply:
I might do something like that if/when I support permissions.